securing_apps/topics/oidc/java/servlet-filter-adapter.adoc

```diff
@@ -64,3 +64,91 @@ To use this filter, include this maven artifact in your WAR poms:
     <version>{project_versionMvn}</version>
 </dependency>
 ----
+ifeval::[{project_community}==true]
+===== Using on OSGi
+
+The servlet filter adapter is packaged as an OSGi bundle, and thus is usable in a generic OSGi environment with HTTP Service and HTTP Whiteboard.
+
+====== Installation
+
+The adapter and its dependencies are distributed as Maven artifacts, so you'll need either working Internet connection to access Maven Central, or have the artifacts cached in your local Maven repo.
+
+If you are using Apache Karaf, you can simply install a feature from the Keycloak feature repo:
```

Comment:
Please add a note for Karaf with Pax-Web users to prefer using Fuse 6/7 adapters (Fuse is based on Karaf) (and refer the relevant sections in this guide).